tests/conftest.py

```diff
@@ -271,7 +278,13 @@ def __init__(self, name=None, imgname=None, keeptb=False, fakeplatform=None):
     def destroy(self):
         if self.appldb:
             del self.appldb
-        if self.cleanup:
+        # In case persisten dvs was used removed all the extra server link
+        # that were created
+        if self.persistent:
+            for s in self.servers:
+                s.destroy()
```

Comment:
I'm confused by this part. 

It makes sense if we have <32 servers that we need to create more, but I don't see that done anywhere?

Also, we destroy all the servers, but do we need to recreate the originals before exiting?

Reply:
> I'm confused by this part.
> 
> It makes sense if we have <32 servers that we need to create more, but I don't see that done anywhere?
@daall 
Caller is this code. Ref: https://github.com/Azure/sonic-swss/blob/541bffbaa9f35236e6925087a7c16b6c68d6f324/tests/conftest.py#L227

Extra port are created in else condition.
Ref: https://github.com/Azure/sonic-swss/blob/541bffbaa9f35236e6925087a7c16b6c68d6f324/tests/conftest.py#L115

> Also, we destroy all the servers, but do we need to recreate the originals before exiting?
@daall if we server port created as part of persistent dvs we don't remove them. cleanup flag is False for those
Ref: https://github.com/Azure/sonic-swss/blob/541bffbaa9f35236e6925087a7c16b6c68d6f324/tests/conftest.py#L113

Reply:
cool, thanks for the thorough explanation! :)